tfjs-backend-wasm/scripts/test-bundle-size.js

```diff
@@ -36,9 +36,7 @@ exec(
 
 shell.cd(dirName);
 shell.cd(wasmDirName);
-exec(
-    `yarn && yarn build-core && ./scripts/build-ci.sh && yarn rollup -c`,
-    {silent: false});
+exec(`yarn && ./scripts/build-ci.sh && yarn rollup -c`, {silent: false});
```

Comment:
This also has to build dependencies, so `yarn && yarn build-deps-ci && yarn build-ci && yarn rollup -c`